renderer/src/components/ActivitiesSection/ActivitiesSection.tsx

```diff
@@ -98,6 +107,17 @@ export default function ActivitiesSection({
     };
   }, []);
 
+  const updateDownloadClickHandler = () => {
+    global.ipcRenderer.send(
+      "redirect",
```

Comment:
I guess, It'd be nice to use a constant for 'redirect' word and replace it everywhere, where is used.